<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>0</y>
      <width>371</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Selecciona dia de la setmana</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>Monday</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tuesday</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wednesday</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Thursday</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Friday</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Saturday</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sunday</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Entra contaminació de CO2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_CO2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Entra contaminació de NO2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_NO2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Punt de mesura</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>Palau Reial</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dimarts</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sants</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Resultats</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Ruta del executable JAR</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPlainTextEdit" name="executableTextEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Envia</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFitxer">
    <property name="title">
     <string>Fit&amp;xer</string>
    </property>
    <addaction name="actionSurt"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>A&amp;juda</string>
    </property>
    <addaction name="actionQuant_a"/>
   </widget>
   <addaction name="menuFitxer"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSurt">
   <property name="text">
    <string>&amp;Surt</string>
   </property>
  </action>
  <action name="actionQuant_a">
   <property name="text">
    <string>&amp;Quant a...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionSurt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
